examples/rcpsp_multiskill/mslib/mslib_parse_and_solve.py

```diff
@@ -32,5 +38,38 @@ def example_parsing_and_local_search():
     plt.show()
 
 
+def example_mslib_cpsat():
+    files_dict = get_data_available()
+    file = [f for f in files_dict["MSLIB4"] if "MSLIB_Set4_1003.msrcp" in f][0]
+    model = parse_file_mslib(file, skill_level_version=False)
+    cp_model = CPSatMSRCPSPSolver(
+        problem=model,
+    )
+    cp_model.init_model(
+        one_worker_per_task=False, slack_skill=False, one_skill_per_task=True
+    )
+    cp_model.cp_model.Minimize(cp_model.variables["makespan"])
```

Comment:
Why this line? This already done in cp_model.init_model() I think ?
And if the goal was to show how to change the objective, why not use rather the higher level api: cp_model.set_model_objective("makespan")?